resources/files/sysv-wrapper.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+unset LD_LIBRARY_PATH
+unset LD_PRELOAD
```

Comment:
Ah, yup. That's another special case like AIX that I didn't even think of. I'll do some research on the OSX linker variables